applications/GeoMechanicsApplication/custom_elements/axisymmetric_stress_state.cpp

```diff
@@ -0,0 +1,68 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Marjan Fathian
+//
+
+#include "axisymmetric_stress_state.h"
+#include "custom_utilities/element_utilities.hpp"
+
+namespace Kratos
+{
+
+static const SizeType NumberOfUDofPerNode = 2; // Axi-symmetric stress state assumes a 2D problem definition
+
+Matrix AxisymmetricStressState::CalculateBMatrix(const Matrix&         GradNpT,
+                                                       const Vector&         Np,
+                                                       const Geometry<Node>& rGeometry) const
+{
+    const double radius = GeoElementUtilities::CalculateRadius(Np, rGeometry);
```

Comment:
I would prefer to use `auto` here as well:
```suggestion
    const auto radius = GeoElementUtilities::CalculateRadius(Np, rGeometry);
```

Reply:
Done